docs/exodus-intro.md

```diff
@@ -23,16 +23,16 @@ It uses:
 * Parallelism - can run compilations and tests in parallel.
 * Hermeticity - builds can access only what is declared and pre-fetched, so they run 'hermetically' without unnecesary noise from additional files. 
 
-### Multi-language environment
+### Multi-language environment 
 Here at Wix, we support different software languages and it's a huge advantage to be able to build multiple languages simultaneously. 
 
-### Scalability
+### Scalability 
 Bazel can handle multiple repositories or any size mono-repo. We are able to scale our codebase and our continuous integration system.
 
-### Extend languages and platforms
+### Extend languages and platforms 
 Bazel enables us to extend to more languages and platforms using Bazel's familiar extension language. Because there is a growing community of Bazel uses, we can share and re-use language rules written by other teams.
```

Comment:
we maintain and contrbute to rules_scala  https://github.com/bazelbuild/rules_scala
there is a big thriing comunity of rules contributers